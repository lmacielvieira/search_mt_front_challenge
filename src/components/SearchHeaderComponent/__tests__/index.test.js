import React from 'react'
import {shallow} from 'enzyme'
import {SearchHeaderComponent} from '../index'

describe('<SearchHeaderComponent />', () => {
  it('renders <SearchHeaderComponent />', () => {
    const wrapper = shallow(<SearchHeaderComponent />)
    expect(wrapper).toMatchSnapshot()
  })
})
