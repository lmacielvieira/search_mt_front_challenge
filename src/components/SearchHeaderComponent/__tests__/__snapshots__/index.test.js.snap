// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchHeaderComponent /> renders <SearchHeaderComponent /> 1`] = `ShallowWrapper {}`;
